### PR TITLE
invoice status

### LINKED ISSUE #64: mark invoices as "paid"
**Is it a bug or a feature request?**

- [ ] It's a bug.
- [x] I'm requesting a new feature.

**Describe your issue**

I pretty much only care about "pending" invoices, for which I haven't been paid. It would be nice to split the invoices into 2 sections: "pending" and "paid"

### PR DESCRIPTION
**Pull request by mathieudutour**
fix #64

I haven't changed the list UI depending on the status. It should probably be tackle with #14

### CODE REVIEW

**Comment by hql287** [comment]
@mathieudutour Thanks for the PR. 👍 
Will you be able to add a few unit tests for this? Perhaps https://github.com/hql287/Manta/pull/68 as well? Looks like the build is broken right now. 

And yes, this needs a UI fix, will look into #14 soon.

**Comment by mathieudutour** [comment]
won't have time to work on it for the next 2 weeks sorry. Feel free to finish the PR, you should have the rights to do so on my fork

File: app.js [hunk 1]
```diff
@@ -132,8 +132,8 @@ function createPreviewWindow() {
 }
 
 function addDevToolsExtension() {
-  BrowserWindow.addDevToolsExtension(process.env.REACT_DEV_TOOLS_PATH);
-  BrowserWindow.addDevToolsExtension(process.env.REDUX_DEV_TOOLS_PATH);
+  if (process.env.REACT_DEV_TOOLS_PATH) BrowserWindow.addDevToolsExtension(process.env.REACT_DEV_TOOLS_PATH);
+  if (process.env.REDUX_DEV_TOOLS_PATH) BrowserWindow.addDevToolsExtension(process.env.REDUX_DEV_TOOLS_PATH);
```
Comment (SkyzohKey) [inline app.js]: What the point of calling this 2 times on 2 same conditions ? o.O

Comment (SkyzohKey) [reply app.js]: Oh ok, didn't seen the nuance (react/redux). Feel free to ignore...

**Comment by hql287** [comment]
@mathieudutour No worries. I'll take it from here. Thanks! :)

**Comment by hql287** [comment]
Closing this. Please refer to this PR instead https://github.com/hql287/Manta/pull/102

**Pull request closed without merging**